core/templates/components/forms/forms-templates/mark-all-audio-and-translations-as-needing-update-modal.component.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for
+ *  MarkAllAudioAndTranslationsAsNeedingUpdateModalComponent.
+ */
```

Comment:
Our `fileoverview` docs should be full sentences. Like "Component for marking all audio and translations...". Same for other files.

Reply:
okay Done